### PR TITLE
Adds typescript support

### PR DESCRIPTION
**Pull request by johnhamm**
Adds ts-node as a dependency and supplied "--require" to the command args of node

### CODE REVIEW

File: src/adapter.ts [hunk 1]
```diff
@@ -111,18 +122,18 @@ export class JasmineAdapter implements TestAdapter, IDisposable {
...
+			const args = [config.configFilePath, JSON.stringify(this.log.enabled)];
 			const childProcess = fork(
 				require.resolve('./worker/loadTests.js'),
 				args,
 				{
 					cwd: this.workspaceFolder.uri.fsPath,
 					env: config.env,
 					execPath: config.nodePath,
-					execArgv: [],
+					execArgv: ["--require", "ts-node/register"],
```
Comment (IvanRave) [inline src/adapter.ts]: Thanks for your commit! I think some developers don't use typescript, some developers uses another version of ts-node. If it possible, we can specify this array of arguments in config.nodeArgv and simply attach it to execArgv

**Comment by johnhamm** [comment]
I went ahead and added the config.nodeArgv settings and removed ts-node from package.json.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Property                           | Description
 `jasmineExplorer.config`           | The location of the Jasmine config file (relative to the workspace folder) (default: `spec/support/jasmine.json`)
 `jasmineExplorer.env`              | Environment variables to be set when running the tests
 `jasmineExplorer.nodePath`         | The path to the node executable to use. By default it will attempt to find it on your PATH, if it can't find it or if this option is set to `null`, it will use the one shipped with VS Code
+`jasmineExplorer.nodArgv`          | The arguments to the node executable to use.  To enable automatic typescript, import ts-node package and use these arguments: [\"--require\", \"ts-node/register\"]
```
Comment (IGx89) [inline README.md]: Looks like a typo here -- should be `jasmineExplorer.nodeArgv`, not `jasmineExplorer.nodArgv`

Comment (johnhamm) [reply README.md]: Nice catch.

**Comment by johnhamm** [comment]
@IvanRave noticed the pull request was approved.  Is there any more steps I need to take on my end (first pull request I've done!)?

**Comment by hbenl** [comment]
Thanks, guys! I have just published v1.2.0 containing your changes.

**Pull request merged**